opba-api-security/src/main/java/de/adorsys/opba/api/security/filter/OperationTypeFilter.java

```diff
@@ -0,0 +1,41 @@
+package de.adorsys.opba.api.security.filter;
+
+import de.adorsys.opba.api.security.config.OperationTypeProperties;
+import de.adorsys.opba.api.security.domain.HttpHeaders;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.FilterChain;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@Slf4j
+@Order(0)
+@Component
+@RequiredArgsConstructor
+public class OperationTypeFilter extends AbstractSecurityFilter {
```

Comment:
It is better to have one security filter - to have core security concept in one place (at least while class size is acceptable)

Reply:
Second filter and abstract one were removed